### PR TITLE
Create privacy-art-collection.md

### PR DESCRIPTION
**Pull request by PhoenixGreen**
Test art collection for privacy

### CODE REVIEW

File: articles/privacy-art-collection.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+Thesis statement for art collection - What the article is about, and what is it aiming to prove
+
+Pre-discussion, research and questions
+
+---
+
+# Article Title
+
+## Art sample + Description
```
Comment (PhoenixGreen) [inline articles/privacy-art-collection.md:25]: Sample Description: Broadly speaking, privacy is the right to be let alone, or freedom from interference or intrusion. Information privacy is the right to have some control over how your personal information is collected and used.

Image:
preserving-privacy-and-security-in-a-generative-ai-world

File: articles/privacy-art-collection.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+|   |   |
+|---|---|
+| Title               | Privacy Art Collection |
+| Author              | OfficialCryptos |
+| Featured text       | FIXME |
```
Comment (PhoenixGreen) [inline articles/privacy-art-collection.md:5]: Need featured test and a better title?

File: articles/privacy-art-collection.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+Pre-discussion, research and questions
+
+---
+
+# Article Title
+
+## Art sample + Description
+
+## Art Piece 1 + Description
```
Comment (OfficialCryptos) [inline articles/privacy-art-collection.md:27]: Decred helps defending your privacy, which is very important when looking for "financial" freedom.
Shared in first place with security in my opinion
decred defending your privacy

Comment (PhoenixGreen) [reply articles/privacy-art-collection.md:27]: perfect 👍

Comment (jpsantanna) [reply articles/privacy-art-collection.md:27]: This art is so good!

File: articles/privacy-art-collection.md (hunk 2 above)
Comment (jpsantanna) [reply articles/privacy-art-collection.md:5]: I like the title. but i'll suggest two more: 1) Veiled Visions: Art Collection on the Essence of Privacy; 2)The Art of Discretion: Exploring Privacy Through Creativity.

Comment (PhoenixGreen) [reply articles/privacy-art-collection.md:5]: I quit like the second "The Art of Discretion: Exploring Privacy Through Creativity."

Comment (OfficialCryptos) [reply articles/privacy-art-collection.md:5]: I like that second title!

File: articles/privacy-art-collection.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+## Art Piece 2 + Description
+
+## Art Piece 3 + Description
```
[the remark this answers is not in the document]
Comment (PhoenixGreen) [reply articles/privacy-art-collection.md:31]: Very cool!

File: articles/privacy-art-collection.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+## Art Piece 4 + Description
```
Comment (OfficialCryptos) [inline articles/privacy-art-collection.md:33]: Decred's coinjoin style privacy system (coinjoin++) is integrated into its governance ticket system.
Participants of the governance system constantly buy tickets time-locking their coins in the process, all those constant transactions gives the decred coinjoin system a much higher anonymity set then most other coinjoin style systems in the market
decred privacy system 3d

**Pull request merged**